Disjoint_Sets/Disjoint_Sets.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+
+public class Disjoint_Sets {
+	static int[] parent;
+	static int[] size;
```

Comment:
@dibyaaaaax I guess the indentation is messed up in the new commit. Please fix this.
You have to add 4 spaces in place of single indentation, 8 spaces in place of double, and so on...
You can do this directly from the IDE / text editor's quick settings.

Reply:
Done.